app/controllers/support_interface/application_forms/email_subscription_controller.rb

```diff
@@ -0,0 +1,36 @@
+module SupportInterface
+  module ApplicationForms
+    class EmailSubscriptionController < SupportInterfaceController
+      before_action :set_application_form
+
+      def edit
+        @email_subscription_form = EmailSubscriptionForm.build_from_application(@application_form)
+      end
+
+      def update
+        @email_subscription_form = EmailSubscriptionForm.new(unsubscribed_from_emails_params)
+
+        if @email_subscription_form.save(@application_form)
+          flash_success
+          redirect_to support_interface_application_form_path(@application_form)
+        else
+          render :edit
+        end
+      end
+
+    private
+
+      def unsubscribed_from_emails_params
+        params.require(:support_interface_email_subscription_form).permit(:unsubscribed_from_emails, :audit_comment)
```

Comment:
```suggestion
        params.expect(support_interface_email_subscription_form: [:unsubscribed_from_emails, :audit_comment])
```

We'r enot enforcing this yet but definitely worth using in new code 👌

Reply:
I'm having a think about this. I want to understand more before I make the change. 

I will deal with it as well as all the others as part of this card: https://trello.com/c/9zs6qRe2/587-tech-debt-enable-new-rubocop-rule-for-paramsexpect